Description: export additional symbols expoted by the public include files
Author: Ludovico Cavedon <cavedon@debian.org>

Index: ndpi/libndpi.sym
===================================================================
--- ndpi.orig/libndpi.sym
+++ ndpi/libndpi.sym
@@ -28,3 +28,6 @@ ndpi_get_proto_breed_name
 ndpi_get_proto_by_id
 ndpi_get_protocol_id_master_proto
 ndpi_guess_protocol_id
+ndpi_get_flow_masterprotocol
+ndpi_protocol2name
+ndpi_set_detected_protocol
